CHANGELOG.md

```diff
@@ -9,17 +9,43 @@ TODO: `cargo`, `std` features
 * **Removed**
   * `From<&yaml_rust::yaml::Hash>` for `ArgGroup`
   * **Macros**
-    * `_clap_count_exprs`
+    * `_clap_count_exprs!`
+    * `arg_enum!` in favor of `ArgEnum` derive macro.
+    * `value_t!` in favor of `ArgMatches::value_of_t`
+    * `value_t_or_exit!` in favor of `ArgMatches::value_of_t_or_exit`
+    * `values_t!` in favor of `ArgMatches::values_of_t`
+    * `values_t_or_exit!` in favor of `ArgMatches::values_of_t_or_exit`
+    * `flags!` in favor of `App::get_flags_no_heading`
+    * `opts!` in favor of `App::get_opts_no_heading`
```

Comment:
While writing this, I realized that we should probably rename the above 2 to `get_flags_without_heading` and `get_opts_without_heading`.

Reply:
1. They weren't internal but exported even though they were hidden.
2. `_without_heading` sounds more readable than `_no_heading`. Alternative `_with_no_heading`

Reply:
1. When talking about public (external)/private (internal) distinction, the intent is what matters. They were never supposed to be used by anybody but clap devs (and I doubt they were), so no point in highlighting them in changelog. 

   As a compromise, maybe put them under a separate section "`#[doc(hidden)]` items" and explain that clap no longer relies on hidden public items and exports legit, documented public API instead? Also, it would be good to mention that hidden `App/Arg` fields are no longer a thing, use getters instead.
2. Sounds better for a person reading a book, sure, but programming is a world of shortcuts, acronyms, and grammar violations all around. `without` instead of `no` would serve no purpose: the method would get longer and slightly harder to type, and programmers reading that would see no difference. I just don't see the point and I do see the counterpoint, this is why I disagree.

   Generally I'm open to suggestions of the kind, but I think it's pointless and harmful in this particular case.

Reply:
I honestly didn't understand what the functions were supposed to do based on their name, had to look at the code. Isn't this one of the reason we changed many function names? To make them more readable and understandable? I would really appreciate if we make it `get_opts_with_no_heading`